website/docs/cdktf/create-and-deploy/project-setup.html.md

```diff
@@ -2,80 +2,71 @@
 
 ```
 $ cdktf init --template="typescript" --local
 ```
 
-### Use with Terraform Cloud
+### Use Terraform Cloud as a Remote Backend
 
-If you initialize a project without passing the `--local` flag, we will use your stored [Terraform Cloud](https://www.terraform.io/docs/cloud/index.html) credentials. If you have no stored Terraform cloud credentials, CDKTF asks if you want to login.
+If you don't want to store Terraform state locally, you can configure your app to use Terraform Cloud as a remote backend. Initialize a project without passing the `--local` flag, to prompt CDKTF to use your stored [Terraform Cloud](https://www.terraform.io/docs/cloud/index.html) credentials to create a new workspace for storing state. If you have no stored Terraform Cloud credentials, CDKTF will ask you to login.
 
-The scaffolded project is configured to use [Terraform state](https://www.terraform.io/docs/language/state/index.html) with a [remote backend](https://www.terraform.io/docs/language/settings/backends/remote.html). Where the Terraform operations will happen depends on the configuration of the Terraform Cloud Workspace settings. If the workspace was created as part of the `cdktf init` command, the Terraform operations will be run locally by default. The Terraform Cloud workspace an be configured to use remote operations instead.
+The new scaffolded project is configured to use Terraform state with a [remote backend](https://www.terraform.io/docs/language/settings/backends/remote.html). Where the Terraform operations will happen depends on the configuration of the Terraform Cloud Workspace settings. When created as part of the `cdktf init` command, the Terraform operations will be run locally by default. The Terraform Cloud workspace an be configured to use remote operations instead (see below).
 
 #### Terraform Cloud VCS Integration
 
-Terraform Cloud supports [connecting to VCS providers](https://www.terraform.io/docs/cloud/vcs/index.html).
+Terraform Cloud supports [connecting to VCS providers](https://www.terraform.io/docs/cloud/vcs/index.html). To use the VCS integration, commit the synthesized Terraform config (the `cdktf.out` directory) alongside your code, so that Terraform Cloud can use it to deploy your infrastructure. On the General Settings page of your Terraform Cloud Workspace [set the Terraform Working Directory](https://www.terraform.io/docs/cloud/workspaces/settings.html#terraform-working-directory) to the output directory of the stack you want to deploy. So for example use `cdktf.out/stacks/dev` if your stack is named `dev` (please refer to the [Stacks documentation](/docs/cdktf/stacks.html) for more information on how you can use stacks to separate the state management for multiple environments within an application).
```

Comment:
PS. Do we have a ticket for the stacks page? I recently added it in a PR somewhere, but I don't want it to get lost because I don't know if it was part of the original plan that we built the board on. 

```suggestion
Terraform Cloud supports [connecting to VCS providers](https://www.terraform.io/docs/cloud/vcs/index.html). To use the VCS integration, commit the synthesized Terraform config (the `cdktf.out` directory) alongside your code, so that Terraform Cloud can use it to deploy your infrastructure. On the General Settings page of your Terraform Cloud workspace [set the Terraform working directory](https://www.terraform.io/docs/cloud/workspaces/settings.html#terraform-working-directory) to the output directory of the stack you want to deploy. So for example, use `cdktf.out/stacks/dev` if your stack is named `dev`. Refer to the [Stacks documentation](./stacks.html) for more information about using stacks to separate the state management for multiple environments in an application.
```

Reply:
@laurapacilio We plan to cover apps and stacks in the architecture page: https://github.com/hashicorp/terraform-cdk/issues/972